game/src/game/components/GuiExampleComponent.h

```diff
@@ -27,4 +28,6 @@ class GuiExampleComponent final : public Component,
   private:
     jss::object_ptr<Transform> transform_;
     jss::object_ptr<InputService> input_service_;
+    jss::object_ptr<PhysicsService> physics_service_;
+    InputService input_service_2_;
```

Comment:
By defining a variable like this, you're making a copy of the InputService, which we don't wanna do. Services should never be copied, cause there can only be one in existence at once, or we'll get issues